bluetooth/src/androidLibMain/kotlin/BluetoothMonitor.kt

```diff
@@ -49,16 +50,17 @@ actual class BluetoothMonitor internal constructor(
         get() = bluetoothAdapter?.isEnabled == true
 
     override fun startMonitoring() {
-        if (bluetoothAdapter == null) return
+        if (bluetoothAdapter == null) warn("BluetoothMonitor") { "bluetoothAdapter is null" }
```

Comment:
Does it return from function?

Reply:
Also once it's subclass of ServiceMonitor you can call super here and in stopMonitoring as well

Reply:
No it does not return. I think it should also print something because otherwise you don't understand why is not starting thee monitoring

Reply:
`bluetoothAdapter` should not be optional, the class should throw an exception when being created.

Reply:
530eca6